package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "dotenv-azure",
-  "version": "2.0.0",
+  "version": "2.0.5",
```

```suggestion
  "version": "2.0.0",
```

You can undo this change, I will bump the version in a separate commit before uploading to npm. Also, this would be a minor bump since it is a new feature.